### PR TITLE
fix: Resolve title toggle issue

### LINKED ISSUE #75: Unintentional toggle between homepage/myrecipes heads when clicking on HOME button

### PR DESCRIPTION
**Pull request by jsahim**
Files changed: scripts.js
Change Summary: Resolved issue where Home logo and user recipes title display were toggling even when home button is clicked, even when the user was already at home view.

This was resolved by adding a conditional  to the goHome function that only allows the goHome function to run if the currentView value was anything other than the all recipes (home view).

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -157,11 +157,13 @@ function showSavedRecipes() {
...
 
 function goHome() {
-  toggleHidden(myRecipesTitle);
-  toggleHidden(titleLogo);
-  savedView = false;
-  currentView = recipeRepo.recipes;
-  searchBar.placeholder = 'Search Recipes...';
-  filterHeader.innerText = 'Filter Recipes';
-  createRecipeCards(currentView);
+  if(currentView !== recipeRepo.recipes){
```
Comment (qrispi) [inline src/scripts.js:160]: Intuitive and clean! Love this fix.

**Review by qrispi — APPROVED** [review]
Great job on this fix!

File: src/scripts.js (hunk 1 above)
Comment (jfogiato) [reply src/scripts.js:160]: +1 to Chris' comment. Looks great!

Comment (LordSchwifty) [reply src/scripts.js:160]: Very clean.Nice fix!

**Review by LordSchwifty — APPROVED** [review]

**Pull request merged**